src/cljs/reagent_toolbox_docs/components/app_bar.cljs

```diff
@@ -1,9 +1,138 @@
 ;;;; Copyright © 2017 Flexpoint Tech Ltd
 
 (ns reagent-toolbox-docs.components.app-bar
-  (:require [reagent-toolbox-docs.layout :as layout]))
+  (:require [reagent-toolbox-docs.ui :as ui]
+            [reagent-toolbox-docs.layout :as layout]))
 
-(defmethod layout/pages :app-bar [_]
-  (fn [_]
+(defn view []
+  (fn []
     [:article
-     [:h1 "App Bar"]]))
+     [:h1 "App Bar"]
+     [:p "The app bar is a special kind of toolbar that’s used for branding, navigation, search, and actions. Usually it contains controls on the right and left side and a title with the current section or app name. You should give the content with children elements."]
+     [:code
+      [ui/display-and-eval-code "
+(do
```

Comment:
It is just a temporary workaround to get the exemple to work, there seems to be a bug with multiple top-level forms in display-and-eval-code.